makelib/golang.mk

```diff
@@ -129,6 +129,8 @@ endif
 GO_COMMON_FLAGS = $(GO_BUILDFLAGS) -tags '$(GO_TAGS)'
 GO_STATIC_FLAGS = $(GO_COMMON_FLAGS) $(GO_PKG_STATIC_FLAGS) -installsuffix static  -ldflags '$(GO_LDFLAGS)'
 GO_GENERATE_FLAGS = $(GO_BUILDFLAGS) -tags 'generate $(GO_TAGS)'
+GO_COVER_MODE ?= count
```

Comment:
i would recommend putting this up near the top of this file, around line 31 that says `# Optional build flags passed to go tools`.  That will help people more easily discover the set of config options that they can set.

nit: consider calling `CGO_ENABLED` -> `GO_CGO_ENABLED` instead, for someone doing a search for `^GO_` can find it along with the other options that are named similarly.

Reply:
Thanks, I followed the advice.